frontend/src/metabase/components/EntityMenu/EntityMenu.jsx

```diff
@@ -92,11 +92,13 @@ class EntityMenu extends Component {
           {menuItemContent || (
             <ol className={CS.p1} style={{ minWidth: minWidth ?? 184 }}>
               {items.map(item => {
+                const key = item.key ?? item.title;
```

Comment:
Things are failing because these `extraButtons` don't have `title` in their definitions:
https://github.com/metabase/metabase/blob/bc88692513d2ad8da0d266d5544f07e56954bcac/enterprise/frontend/src/metabase-enterprise/audit_app/index.js#L24-L51